gateway/assets/js/faas-ui.js

```diff
@@ -0,0 +1,23 @@
+angular
+  .module('faasUI', ['ngMaterial'])
+  .controller('faasCtrl', function($mdSidenav, $scope) {
+    $scope.avatar = function(func) {
+      return 'https://www.gravatar.com/avatar/' + md5(func.name) + '?d=identicon&f=y';
+    };
+    $scope.functions = [
```

Comment:
Why is this hard-coded list of functions here?

Reply:
That was a WIP for testing a different strategy for state management. Since this is out of the scope, you can forget about it.